core/utils/cutils.class.php

```diff
@@ -68,4 +68,38 @@ public static function format_Number($number, $decimal = 0)
         // Return formated number
         return number_format($number, $decimal, $locale['decimal_point'], $locale['thousands_sep']);
     }
+
+    // ==================================================================================
+    // Function: 	format_Date()
+    // Parameters:	$date
+    //				$format (optional, default = null)
+    // Return:		formated date
+    // ==================================================================================
+    public static function format_Date($date, $format = null)
```

Comment:
Why create a format_date() function in cutils.class.php when we have a DateTimeUtil in datetimeutil.class.php ?